l10n_br_sale/models/sale_order_line.py

```diff
@@ -122,19 +122,17 @@ def _onchange_product_uom(self):
     def _onchange_discount_percent(self):
         """Update discount value"""
         if not self.env.user.has_group('l10n_br_sale.group_discount_per_value'):
-            if self.discount:
-                self.discount_value = (
-                    (self.product_uom_qty * self.price_unit)
-                    * (self.discount / 100)
-                )
+            self.discount_value = (
+                (self.product_uom_qty * self.price_unit)
+                * (self.discount / 100)
+            )
 
     @api.onchange('discount_value')
     def _onchange_discount_value(self):
         """Update discount percent"""
         if self.env.user.has_group('l10n_br_sale.group_discount_per_value'):
-            if self.discount_value:
-                self.discount = ((self.discount_value * 100) /
-                                 (self.product_uom_qty * self.price_unit))
+            self.discount = ((self.discount_value * 100) /
```

Comment:
Não precisa verificar se existe o valor do desconto ?

Reply:
No, if it's zero the result will be correct.

Reply:
@gabrielcardoso21 para atingir o objetivo de cobertura de teste, consegue implementar um teste para esta linha 134 ?